job-dsls/jobs/kie/branch/daily_builds.groovy

```diff
@@ -5,7 +5,7 @@ def MVN_TOOL=Constants.MAVEN_TOOL
 def KIE_VERSION=Constants.KIE_PREFIX
 def BASE_BRANCH=Constants.BRANCH
 def GH_ORG_UNIT=Constants.GITHUB_ORG_UNIT
-def AGENT_LABEL="rhos-01 && kie-rhel7-pipeline && kie-mem24g && !built-in"
+def AGENT_LABEL="rhos-01 && kie-rhel8-pipeline && kie-mem24g && !built-in"
```

Comment:
this should be available no need to change see https://gitlab.cee.redhat.com/business-automation/eng-jenkins/-/blob/b95cedb48b3c856abc752a74bb621b38cea4e473/casc.yaml#L441